bar-trender-front-end/src/components/Modals/ModalSelectedElement.js

```diff
@@ -24,19 +24,20 @@ function ModalSelectedElement(prop) {
   const [modal1, setModal1] = React.useState(false);
   const [modal2, setModal2] = React.useState(false);
   const { element } = prop;
+  // console.log(element, 'element');
   const [appState, setAppState] = useState({
     discounts: {},
   });
 
    useEffect(() => {
-    const apiUrl = "https://develop-backend-sprint-01.herokuapp.com/v1/establishments/"+element.id+"/discounts/get";
+    const apiUrl = "https://develop-backend-sprint-01.herokuapp.com/v1/establishments/"+element.id+"/discounts/get?page=1&all=False";
     async function loadDiscounts(){
       await fetch(apiUrl, {
         method: 'GET',
         headers: {
           'Content-Type': 'application/json',
           'token': '',
-
+          'apiKey': '8dDc431125634ef43cD13c388e6eCf11',
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/BarTrender/BarTrenderFront/pullRequest?prid=7167715)
```suggestion
          "apiKey": '8dDc431125634ef43cD13c388e6eCf11',
```